### PR TITLE
inverting the logic to favor newer versions of electrs

### LINKED ISSUE #50: Error: An unknown argument '--jsonrpc-import' was specified when used with locally installed electrs v0.9.7
Now that it is more likely to have an electrs version higher than v0.9.1, it might make sense to invert the logic on:
https://github.com/RCasatta/electrsd/blob/master/src/lib.rs#L225

### PR DESCRIPTION
**Pull request by ulrichard**

### CODE REVIEW

**Comment by ulrichard** [comment]
Tested with locally installed binaries:

```
$ /usr/bin/bitcoind --version
Bitcoin Core version v22.0.0
$ /usr/bin/electrs --version
v0.9.7
$ELECTRS_EXE=/usr/bin/electrs BITCOIND_EXE=/usr/bin/bitcoind RUST_LOG=debug cargo test
```

File: src/lib.rs [hunk 1]
```diff
@@ -77,8 +76,17 @@ pub struct Conf<'a> {
 
 impl Default for Conf<'_> {
     fn default() -> Self {
+        let args = if cfg!(feature = "electrs_0_9_1")
+            || cfg!(feature = "electrs_0_8_10")
+            || cfg!(feature = "esplora_a33e97e1")
+        {
+            vec!["-vvv"]
+        } else {
+            vec!["--log-filters", "debug"]
```
Comment (ulrichard) [inline src/lib.rs:85]: maybe "info" would be enough, as it is really quite verbose like this.

Comment (RCasatta) [reply src/lib.rs:85]: Yeah, makes sense to remove it if it works through the RUST_LOG env var

**Comment by RCasatta** [comment]
Concept ACK

**Pull request merged**